third_party/amd/include/Dialect/TritonAMDGPU/IR/TritonAMDGPUOps.td

```diff
@@ -109,13 +109,15 @@ def BufferLoadOp : TT_AMDGPU_Op<"buffer_load", [
       ins
       TT_Ptr:$ptr,
       I32Tensor:$offsets,
+      DefaultValuedAttr<TT_CacheModifierAttr, "::mlir::triton::CacheModifier::NONE">:$cache,
```

Comment:
Do we have a mapping for all cache modifiers supported in the frontend? If not should we add verifier to only accept those we can handle?

Reply:
From what I understand we can support all modifiers. Am I correct @yiqian1 ?

Reply:
Yes, we can support all cache modifiers.